dGame/dComponents/RacingControlComponent.cpp

```diff
@@ -374,8 +374,14 @@ void RacingControlComponent::HandleMessageBoxResponse(Entity* player, int32_t bu
 		data->collectedRewards = true;
 
 		// Calculate the score, different loot depending on player count
-		const auto score = m_LoadedPlayers * 10 + data->finished;
-
+		auto playersRating = m_LoadedPlayers;
+		if(m_LoadedPlayers = 1){
+			if (m_SoloRacing) {
```

Comment:
This can be simplified to if (m_LoadedPlayers == 1 && m_SoloRacing) playersRating *= 2;

Also you did an assignment in the if statement here, not a comparison.  Double also, make sure to run the editorconfig (alt+shift+f) after working on a file.

Reply:
How is it now?